server/libs/infra/src/db/entities/asset.entity.ts

```diff
@@ -71,7 +71,7 @@ export class AssetEntity {
 
   @Column({ type: 'bytea', nullable: true, select: false })
   @Index({ where: `'checksum' IS NOT NULL` }) // avoid null index
```

Comment:
Removing null doesn't make the column nullable, you have change that in the `@Column` decorator and for this specific instance the `@Index` also. But is this change backwards compatible?

Reply:
Thanks, can you look at the new version please?

Reply:
Yes that's good, you can also remove the `@Index` because checksum can never be null

Reply:
Sorry, I meant to say remove the where part, so you just have `@Index()`. There is still a composite index generated for ownerId and checksum, but it's probably safer to add the index back.

Reply:
After a discussion in discord I think we keep the composite index only